gff/BCBio/GFF/GFFParser.py

```diff
@@ -69,7 +77,7 @@ def _split_keyvals(keyval_str):
 
         GFF3 has key value pairs like:
           count=9;gene=amx-2;sequence=SAGE:aacggagccg
-        GFF2 and GTF have:           
```

Comment:
If you don't use auto-formatting your PR would be easier to remove! Thanks for giving this a stab!

Reply:
(s/remove/review/ presumably)

Reply:
yeah, sorry, I figured auto-formatting is enabled everywhere by default now anyway, and the diff is not so huge